python/py-tmuxp/Portfile

```diff
@@ -61,22 +52,38 @@ if {${name} ne ${subport}} {
              ${worksrcpath}/${subport}
     }
 
-    post-build {
-        system -W ${worksrcpath}/docs "make PYTHONPATH=${worksrcpath}/build/lib SPHINXBUILD=${prefix}/bin/sphinx-build-${python.branch} text man"
+    pre-destroot {
+        xinstall -d ${destroot}${prefix}/share/examples/${subport}
+        xinstall -m 644 {*}[glob ${worksrcpath}/examples/*] \
+            ${destroot}${prefix}/share/examples/${subport}
     }
 
-    post-destroot {
-        set doc_build_dir ${worksrcpath}/docs/_build
+    variant html_man_docs \
+        description {Build and install documentation in HTML and manpage format} {
+        depends_build-append \
+                    port:poetry \
+                    port:py${python.version}-alagitpull \
+                    port:py${python.version}-docutils \
+                    port:py${python.version}-myst-parser \
+                    port:py${python.version}-recommonmark \
+                    port:py${python.version}-sphinx \
+                    port:py${python.version}-sphinx-autodoc-typehints \
+                    port:py${python.version}-sphinx-issues
 
-        xinstall -m 644 {*}[glob ${doc_build_dir}/text/*.txt] \
-            ${destroot}${prefix}/share/doc/${subport}
+        post-build {
+            system -W ${worksrcpath}/docs "make PYTHONPATH=${worksrcpath}/build/lib SPHINXBUILD=${prefix}/bin/sphinx-build-${python.branch} html man"
+        }
+        
+        post-destroot {
```

Comment:
I am not 100% sure, but perhaps you could check this... If the variant is specified you define `post-destroot` but this was already defined in the "main" port to install the examples. I *think* you are overriding it here and, therefore, not install the examples anymore.

Reply:
I also thought about this, so I put examples in `pre-destroot`.

Reply:
It is fine to have multiple instances of post-destroot etc. defined as they all get executed in the order they are defined.

Reply:
Thank you, in that case, I will prefer `post-destroot`, since moving the examples does not look like an operation that should be done before `destroot`.

Reply:
thanks Chris for the info, I didn't realize that... Is that the case for all "pre" and "post" phases, in contrast to, for example, "build" or "destroot"?